mlflow/pipelines/regression/v1/pipeline.py

```diff
@@ -96,8 +96,13 @@
                 always registered with the MLflow Model Registry when the **register** step is
                 executed.
 
-    - **predict**
-      - The **predict** step
+   - **predict**
+      - The **predict** step uses the ingested dataset for scoring created by the
+        **ingest_scoring** step and applies the specified model to the dataset.
```

Comment:
On L17, it should be
```
**ingest_scoring** -> **predict**
```
Also, can you add an entry for step `**ingest_scoring**` before L99?